### DISCUSSION TITLE
How to handle routing in NextJS for blog posts that are part of a series?

### DISCUSSION THREAD

**Discussion by rockchalkwushock** [vercel/next.js#15372] [Help] [answered]
I'm stumped on how to do this. At the moment I have similar routing as the example starters:

```txt
pages/blog/[slug].tsx --> https://.../blog/my-post
```

What I'd like to do is if it is a series post:

```txt
/blog/[series-name]/[slug].tsx

## TOC
1. Getting Started  ---> https://.../blog/my-series/getting-started
2. yadda yadda yadda ---> https://.../blog/my-series/yadda-yadda-yadda
```

Has anyone done this or have any ideas?

**Comment by leerob** [comment, marked as answer]
You could try catch-all routes -> https://nextjs.org/docs/routing/dynamic-routes#catch-all-routes

**Reply by rockchalkwushock** [reply]
@leerob That will work out perfectly! Thanks 🎉
